dataactbroker/helpers/fabs_derivations_helper.py

```diff
@@ -196,30 +190,46 @@ def derive_ppop_location_data(obj, sess, ppop_code, ppop_state_code, county_dict
             obj['place_of_perform_county_co'] = city_info.county_number
             obj['place_of_perform_county_na'] = city_info.county_name
 
-            if (obj['place_of_performance_zip4a'] and obj['place_of_performance_zip4a'].upper() == 'CITY-WIDE'):
-                obj['place_of_performance_scope'] = "City-wide"
-            else:
-                obj['place_of_performance_scope'] = None
-        elif re.match('^[A-Z]{2}\d{4}R$', ppop_code):
-            if (obj['place_of_performance_zip4a'] and obj['place_of_performance_zip4a'].upper() == 'CITY-WIDE'):
-                obj['place_of_performance_scope'] = "City-wide"
-            else:
-                obj['place_of_performance_scope'] = None
-        elif re.match('^[A-Z]{2}\*{5}$', ppop_code):
-            obj['place_of_performance_scope'] = "State-wide"
-        elif re.match('^00\*{5}$', ppop_code):
-            obj['place_of_performance_scope'] = "Multi-state"
-        elif ppop_code == '00FORGN':
-            obj['place_of_performance_scope'] = "Foreign"
-        else:
-            obj['place_of_performance_scope'] = None
-
     # if there's no ppop code, just set them all to None
     else:
         obj['place_of_perform_county_co'] = None
         obj['place_of_perform_county_na'] = None
         obj['place_of_performance_city'] = None
-        obj['place_of_performance_scope'] = None
+
+
+def derive_ppop_scope(obj):
+    """ Deriving place of performance scope values from zip4 and place of performance code
+
+        Args:
+            obj: a dictionary containing the details we need to derive from and to
+    """
+    derived_ppop_scope = None
```

Comment:
Why are we making this a variable instead of just setting it?

Reply:
For me it's easier to see `derived_ppop_scope =` multiple times instead of `obj['place_of_performance_code'] =` multiple times when really the function is only calling it once.

Reply:
Is this the direction we're going with this file from now on? All other instances in it where there are multiple options simply set it.

Reply:
It's something I'd prefer but it doesn't have to be one or the other.